### ISSUE TITLE
Regarding Ciphertext Size

### ISSUE THREAD

**Issue by YongGuCheng** [Di-Chai/he-benchmark#2]
Thank your for sharing.

However, I have difficulty in understanding the Ciphertext Size. 
How did you compute the Ciphertext Size? For example, for Paillier scheme with keysize = 1024, how did you get the Ciphertext Size of 0.0005MB.

**Comment by Di-Chai** [comment]
Hi, thanks for raising the concerns.

Briefly, I collected the memory usage by the current process and computed the memory increase before and after encrypting a certain number of plaintexts. Afterward, the average ciphertext size is calculated and reported on the README page.

**Comment by YongGuCheng** [comment]
> Hi, thanks for raising the concerns.
> 
> Briefly, I collected the memory usage by the current process and computed the memory increase before and after encrypting a certain number of plaintexts. Afterward, the average ciphertext size is calculated and reported on the README page.

Hi Di, 

Thanks a lot for your reply.

Yet, for Paillier scheme with a key size being 1024 (i.e., with the modulus n being 1024 bits),  why is the ciphertext size of 0.0005MB? 

In theory, for Paillier scheme with a key size being 1024 (i.e., with the modulus n being 1024 bits),  n*n is about 2048 bits,  so the ciphertext size should be 2048/1024/1024/8 = 0.000244MB ~= 0.00025MB (say, including overhead in serializatio).

In other words, the ciphertext size of 0.0005MB should correspond to the Paillier scheme with a key size being 2048 (with n being 2048 bits).  Right?
